database/schema/01-auth.sql

```diff
@@ -1,4 +1,14 @@
 CREATE TABLE account
 (
     address TEXT NOT NULL PRIMARY KEY
+);
+
+
+CREATE TABLE vesting_account
+(
+    address             TEXT        NOT NULL REFERENCES account (address) PRIMARY KEY,
+    original_vesting    JSONB       NOT NULL DEFAULT '{}',
+    end_time            TEXT        NOT NULL,
+    start_time          TEXT        NOT NULL,
```

Comment:
Probably it's better to use a `TIMESTAMP WITHOUT TIMEZONE` instead of a `TEXT` here. That would allow for a better support inside GraphQL too